sound/soc/sof/intel/hda.c

```diff
@@ -612,7 +616,10 @@ static int hda_init(struct snd_sof_dev *sdev)
 	/* HDA bus init */
 	sof_hda_bus_init(bus, &pci->dev);
 
-	bus->use_posbuf = 1;
+	if (sof_hda_position_quirk == SOF_HDA_POSITION_QUIRK_USE_DPIB_REGISTERS)
+		bus->use_posbuf = 0;
+	else
+		bus->use_posbuf = 1;
```

Comment:
this is a change in behaviour for the current mode, is this intended?

Reply:
the commit message is pretty clear, no?